packages/design-system/src/styles/components/_Tooltip.scss

```diff
@@ -18,6 +19,20 @@ $tooltip-background-color-inverse: $color-background !default;
   margin: 0;
 }
 
+%trigger-underline-styles {
+  text-decoration: underline;
+  text-decoration-style: dotted;
+  text-underline-position: under;
```

Comment:
Added `text-underline-position` to unsure the text link it below descending letters. This works more consistently than offset. 

the `under` value Forces the line to be set below the alphabetic baseline, at a position where it won’t cross any descenders. This is useful for ensuring legibility with chemical and mathematical formulas, which make a large use of subscripts. https://developer.mozilla.org/en-US/docs/Web/CSS/text-underline-position#values